crash-effects-backup.js

```
@@ -0,0 +1,358 @@
+        // Create gradient circle
+        const gradient = ctx.createRadialGradient(32, 32, 0, 32, 32, 32);
+        gradient.addColorStop(0, 'rgba(100, 100, 100, 1)');
+        gradient.addColorStop(0.5, 'rgba(80, 80, 80, 0.8)');
+        gradient.addColorStop(1, 'rgba(60, 60, 60, 0)');
+        
+        ctx.fillStyle = gradient;
+        ctx.fillRect(0, 0, 64, 64);
+        
+        this.smokeTexture = new THREE.CanvasTexture(canvas);
+        console.log('Created fallback smoke texture');
+    }
```

Biome (1.9.4)

[error] Duplicate class member name "createFallbackTexture"



(lint/suspicious/noDuplicateClassMembers)